src/cljs/ataru/virkailija/editor/component.cljs

```diff
@@ -631,21 +702,11 @@
             [:header.editor-form__component-item-header "Vastausvaihtoehdot"]
             (when-not (= field-type "singleChoice") [dropdown-multi-options path options-koodisto])]
 
-           (when (nil? @options-koodisto)
-             (seq [
-                   ^{:key "options-input"}
-                   [:div.editor-form__multi-options-container
-                    (map-indexed (fn [idx _]
-                                   (dropdown-option idx path languages :include-followup? (and (not question-group-element?)
-                                                                                            (some #{field-type} ["dropdown" "multipleChoice" "singleChoice"]))))
-                      (:options @value))]
-                   ^{:key "options-input-add"}
-                   [:div.editor-form__add-dropdown-item
-                    [:a
-                     {:on-click (fn [evt]
-                                  (.preventDefault evt)
-                                  (dispatch [:editor/add-dropdown-option path]))}
-                     [:i.zmdi.zmdi-plus-square] " Lisää"]]]))]]]))))
+           (if (nil? @options-koodisto)
+             (let [editable? true]
```

Comment:
Tän `editable?`n vois varmaan vaan inlinettää vaikka onkin toki eksplisiittinen näin